### PR TITLE
🎨 [Guide Re-org] - Consolidated `Onboarding` and `Payments` into the `Daily Spending Wallet` reference design

### PR DESCRIPTION
**Pull request by Bosch-0**
Consolidated `Onboarding` and `Payments` into a `Daily spending wallet` reference design section.

Closes https://github.com/BitcoinDesign/Guide/issues/718 
Closes https://github.com/BitcoinDesign/Guide/issues/721 

💻 [Deploy Preview](https://deploy-preview-739--sad-borg-390916.netlify.app/guide/daily-spending-wallet/landing-page/)

### CODE REVIEW

**Comment by pavlenex** [comment]
Screenshot 2022-04-04 at 18 08 28

🤯 This one is going to take a while to review. I assume this is because of all the link changes.

**Review by pavlenex — CHANGES_REQUESTED** [review]
@Bosch-0 Since I'm not sure what was the consensus regarding the implementation, I only reviewed for broken links. 

`Rake test` found 18 broken links. Unfortunatelly do to large amount of changes, my GitHub web crashes, so I cannot recommend them directly, instead here's the log of broken internal links that need to be fixed, luckily the log is quite descriptive on which pages have broken links.

```


Ran on 104 files!


- ./_site/guide/case-studies/cloud-backup/index.html
  *  internally linking to /guide/onboarding/introduction/, which does not exist (line 347)
     <a href="/guide/onboarding/introduction/">Onboarding</a>
- ./_site/guide/case-studies/savings-account/index.html
  *  internally linking to /guide/onboarding/introduction/, which does not exist (line 316)
     <a href="/guide/onboarding/introduction/">onboarding</a>
- ./_site/guide/case-studies/shared-account/index.html
  *  internally linking to /guide/onboarding/introduction/, which does not exist (line 337)
     <a href="/guide/onboarding/introduction/">onboarding</a>
- ./_site/guide/case-studies/upgradeable-wallet/index.html
  *  internally linking to /guide/onboarding/introduction/, which does not exist (line 311)
     <a href="/guide/onboarding/introduction/">onboarding</a>
- ./_site/guide/daily-spending-wallet/backup-and-recovery/manual-backup/index.html
  *  internally linking to /daily-spending-wallet/backup-and-recovery/recovery/, which does not exist (line 688)
     <a class="next-previous--next" href="/daily-spending-wallet/backup-and-recovery/recovery/" rel="next">
      Recovery
      <svg viewBox="0 0 24 24">
        <use xlink:href="#svg-arrow-right"></use>
      </svg>
    </a>
- ./_site/guide/daily-spending-wallet/backup-and-recovery/recovery/index.html
  *  internally linking to /guide/daily-spending-wallet/request, which does not exist (line 366)
     <a href="/guide/daily-spending-wallet/request">requesting</a>
- ./_site/guide/daily-spending-wallet/first-use/index.html
  *  internally linking to /guide/daily-spending-wallet//restoring-a-wallet/, which does not exist (line 295)
     <a href="/guide/daily-spending-wallet//restoring-a-wallet/">Restore an existing wallet</a>
  *  internally linking to /guide/daily-spending-wallet/backing-up-a-wallet/, which does not exist (line 296)
     <a href="/guide/daily-spending-wallet/backing-up-a-wallet/">Backup a wallet</a>
  *  internally linking to /guide/daily-spending-wallet/funding-a-wallet/, which does not exist (line 297)
     <a href="/guide/daily-spending-wallet/funding-a-wallet/">Fund a wallet</a>
- ./_site/guide/daily-spending-wallet/funding/index.html
  *  internally linking to /guide/daily-spending-wallet/backup-and-recovery/, which does not exist (line 754)
     <a href="/guide/daily-spending-wallet/backup-and-recovery/">backups &amp; recovery</a>
  *  internally linking to /guide/daily-spending-wallet/backup-and-recovery/, which does not exist (line 760)
     <a class="next-previous--next" href="/guide/daily-spending-wallet/backup-and-recovery/" rel="next">
      Backing up a wallet
      <svg viewBox="0 0 24 24">
        <use xlink:href="#svg-arrow-right"></use>
      </svg>
    </a>
- ./_site/guide/daily-spending-wallet/requesting/index.html
  *  internally linking to /guide/onboarding/introduction/, which does not exist (line 545)
     <a href="/guide/onboarding/introduction/">onboarding</a>
- ./_site/guide/designing-products/common-user-flows/index.html
  *  internally linking to /guide/onboarding/introduction/, which does not exist (line 572)
     <a href="/guide/onboarding/introduction/">onboarding section</a>
- ./_site/guide/designing-products/usage-life-cycle/index.html
  *  internally linking to /guide/onboarding/introduction/, which does not exist (line 392)
     <a href="/guide/onboarding/introduction/">onboarding experience</a>
- ./_site/guide/designing-products/wallet-interoperability/index.html
  *  internally linking to /guide/onboarding/introduction/, which does not exist (line 528)
     <a href="/guide/onboarding/introduction/">onboarding</a>
  *  internally linking to /guide/onboarding/introduction/, which does not exist (line 534)
     <a class="next-previous--next" href="/guide/onboarding/introduction/" rel="next">
      Onboarding
      <svg viewBox="0 0 24 24">
        <use xlink:href="#svg-arrow-right"></use>
      </svg>
    </a>
- ./_site/guide/getting-started/principles/index.html
  *  internally linking to /guide/onboarding/introduction/, which does not exist (line 378)
     <a href="/guide/onboarding/introduction/">onboarding</a>
- ./_site/guide/glossary/address/index.html
  *  internal image /assets/images/guide/payments/request/single-use-payment-request-settings-address@2x.png does not exist (line 383)

HTML-Proofer found 18 failures!
```

File: guide/daily-spending-wallet/backup-and-recovery/manual-backup.md [hunk 1]
```diff
@@ -313,8 +315,8 @@ You could also ask users to select (or manually type out) a random word from the
 The next section will highlight some options for [protecting a wallet]({{ '/guide/onboarding/protecting-a-wallet/' | relative_url }}).
```
Comment (sbddesign) [inline guide/daily-spending-wallet/backup-and-recovery/manual-backup.md:315]: The next section is now "recovery"

File: guide/daily-spending-wallet/privacy.md [hunk 2]
```diff
@@ -156,11 +158,11 @@ There is no perfect solution to guarantee 100% privacy that lasts forever becaus
 
 ---
 
-Next, let's dive into how to display various [units & symbols]({{ '/guide/payments//units-and-symbols/' | relative_url }}) in bitcoin applications.
+Next, let's dive into how to [secure]({{ '/guide/daily-spending-wallet/security/' | relative_url }}) your bitcoin when using a daily spending wallet.
```
Comment (sbddesign) [inline guide/daily-spending-wallet/privacy.md:161]: The next section is now "case studies"

**Review by sbddesign — DISMISSED** [review]
Looking pretty good. cACK on bundling recovery with the backups in the nav.

Requesting changes:

1. There are two pages footers with outdated descriptions for the next link, left inline comments to point these out.
2. `rake test` has 18 broken internal links, same ones @pavlenex indicated.
3. Daily spending image for intro page would be good, but could be handled in another PR.

Fix 1 and 2 and I'll review it again.

**Comment by Bosch-0** [comment]
Fixed all the above comments, also moved content from https://bitcoin.design/guide/case-studies/cloud-backup/ to this new section and removed that case study. We can re-purpose some of that content when we make a reference design page inside the daily spending wallet section (brings all the chapter together into a prototype).

File: guide/daily-spending-wallet/landing-page.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+This section is an in-depth design exploration of a bitcoin-only, non-custodial, mobile lightning-first, wallet for on-the-go payments.
+
+Imagine a product which tries to solve the problem of quickly and easily sending smaller amounts of money to friends and family, or paying for [small purchases]({{ '/guide/designing-products/personal-finance/#day-to-day-spending' | relative_url }}). Ease and speed of use will be important as usage is likely to be on mobile devices and on the go. Users are not expected to be well-versed in bitcoin technology or advanced private key management, making it reasonable to worry more about self-inflicted loss than theft.
+
+---
+
+### [First use]({{ '/guide/daily-spending-wallet/first-use/' | relative_url }})
+
+An overview of common tasks when using a wallet application for the first time.
```
Comment (GBKS) [inline guide/daily-spending-wallet/landing-page.md:40]: We should rewrite these short descriptions to avoid repeat of "An overview" and "An introduction". Can be a separate PR.

File: guide/how-it-works/introduction.md [hunk 4]
```diff
@@ -2,7 +2,7 @@
 layout: guide
 title: How it works
 description: Find out about the complexities
```
Comment (GBKS) [inline guide/how-it-works/introduction.md:4]: Is this sentence cut off?

File: guide/onboarding/introduction.md [hunk 5]
```diff
@@ -1,85 +0,0 @@
...
-   mobileRetina = "/assets/images/guide/onboarding/onboarding-main-mobile@2x.png"
-   alt-text = "Onboarding landing page header"
-   width = 1600
-   height = 600
-   layout = "full-width"
-%}
-
-# Onboarding
-
-Onboarding, the first use, can make or break someone’s experience of a product. Therefore it is important to identify your target audience and design specifically with its context, expectations, and needs in mind.
```
Comment (GBKS) [inline guide/onboarding/introduction.md:38]: Some good top-level info about onboarding here. Wonder if we could re-use this somewhere instead of deleting it.

**Review by GBKS — CHANGES_REQUESTED** [review]
Huge and tedious update, thanks for working through this. Had some minor comments. If we want to move forward with merging this PR, I'm totally fine with handling text changes in a separate one. 

For future re-orgs like these, let's split things up a bit further. The Github review page was so overloaded that my browser didn't work properly until I managed to use the file filter to only show markdown files. Moving images can happen in separate PRs (maybe even page-by-page to split it up further).

My review included:
- Running the site locally
- rake test
- Going over all markdown files on Github
- Clicking through all pages locally and checking Terminal and Browser Dev Tools for issues

One question I have that I also asked in Slack is about moving images. We want to avoid re-compression of images because that leads to a loss of quality. The local dev environment tracks images it has compressed in `_compress_images_cache.yml`. Do you manually update this file when you move images to avoid re-compression?

File: guide/daily-spending-wallet/backup-and-recovery/recovery.md [hunk 6]
```diff
@@ -18,27 +21,27 @@ Illustration sources:
...
+   mobile = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restoring-a-wallet-mobile.png"
+   mobileRetina = "/assets/images/guide/daily-spending-wallet/backup-and-recovery/recovery/restoring-a-wallet-mobile@2x.png"
    alt-text = "Restoring a wallet chapter header image"
    width = 1600
    height = 600
    layout = "full-width"
 %}
 
-# Restoring a wallet
+# Recovery
```
[the remark this answers is not in the document]
Comment (Bosch-0) [reply guide/daily-spending-wallet/backup-and-recovery/recovery.md:34]: Good point, definitely something to explore further

File: guide/case-studies/savings-wallet.md [hunk 7]
```diff
@@ -1,13 +1,15 @@
...
 layout: guide
-title: Savings account
+title: Savings wallet
 description: A UX case study for a multi-key bitcoin wallet designed for both daily spending and storing medium amounts.
 nav_order: 2
 parent: Case studies
-permalink: /guide/case-studies/savings-account/
+permalink: /guide/case-studies/savings-wallet/
+redirect_from:
+ - /guide/case-studies/multisig-wallet/
```
[the remark this answers is not in the document]
Comment (Bosch-0) [reply guide/case-studies/savings-wallet.md:9]: > Is /guide/case-studies/multisig-wallet/

Yeah it is. My bad on that one, the image folder for this was originally multisig-wallet so got confused with that. fixed.

File: guide/onboarding/introduction.md (hunk 5 above)
Comment (Bosch-0) [reply guide/onboarding/introduction.md:38]: I made backups of all these pages fyi, also saved in Git history but just in case. Was keen to re-purpose a lot of that content.

**Review by GBKS — APPROVED** [review]

**Comment by GBKS** [comment]
ACK. Thanks for addressing all the changes so quickly.

**Review by pavlenex — APPROVED** [review]
LGTM

**Review by pavlenex — APPROVED** [review]

**Comment by Mylenek** [comment]
First use

**Comment by Mylenek** [comment]
Back up and recovery

This particular image communicates the idea well but visually feels imposing due to it's scale and bold color.

**Comment by Mylenek** [comment]
Requesting bitcoin

A perfectly fine graphic that [with a bit of cropping] could become more harmonious with the other visuals in the deck.

**Comment by Mylenek** [comment]
Floating text

Here, I pulled 3 examples of callouts that have "floating" text... there are other instances in this section where this occurs but I'm certain you get the point. Consider adjusting indent or character spacing in order to modify the text, thereby eliminating the extra line [one "floating" word] in the callout.

**Comment by Mylenek** [comment]
FYI - I'm new to GitHub so if there is a better way to communicate or use this platform please let me know and I'll adjust.
THX!

**Comment by GBKS** [comment]
@Mylenek thanks for the feedback. For the point about floating words (aka orphans), that's something we discussed before. It's just one of those things about responsive design where you don't have full control over every detail. Text wraps differently based on screen size and user font settings via the browser or operating system. There's some info here in the [Apple Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/accessibility/overview/text-size-and-weight/).

So manually tweaking text would only fix this for a specific configuration. Two other ways to fix it are via Javascript, like the [NY times text balancer](https://www.ctrl.blog/entry/text-wrap-balance.html), or manually inserting non-breaking spaces between the last two words of every paragraph which is a bit messy. We briefly discussed these, but it's just not been a priority so far. Maybe we need to pick this up again sometime.

**Comment by GBKS** [comment]
Also, good points about the images. Nice thing about open-source is that we can continuously tweak things. I'll create issues for each of these and then we can address them.

**Comment by Mylenek** [comment]
Thanks for taking the time to thoughtfully reply @GBKS I'm learning & appreciate your consideration 👊
